src/client/theme-default/styles/layout.css

```diff
@@ -2,17 +2,17 @@ body {
   padding: 0;
   margin: 0;
   background-color: #fff;
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen,
```

Comment:
this is necessary to type the font of the search modal. Vuepress has the same thing